sdk/identity/Azure.Identity/src/DefaultAzureCredentialFactory.cs

```diff
@@ -34,7 +34,7 @@ public virtual TokenCredential CreateSharedTokenCacheCredential(string tenantId,
 
         public virtual TokenCredential CreateInteractiveBrowserCredential(string tenantId)
         {
-            return new InteractiveBrowserCredential(tenantId, Constants.DeveloperSignOnClientId, new InteractiveBrowserCredentialOptions { TokenCache = new PersistentTokenCache() }, Pipeline);
+            return new InteractiveBrowserCredential(tenantId, Constants.DeveloperSignOnClientId, new InteractiveBrowserCredentialOptions { TokenCachePersistenceOptions = new InMemoryTokenCacheOptions() }, Pipeline);
```

Comment:
This should use `TokenCachePersistenceOptions` if it was using `PersistentTokenCache` no?